test/ini_files/cgap.ini

```diff
@@ -9,6 +9,7 @@ blob_bucket = elasticbeanstalk-fourfront-cgap-blobs
 system_bucket = elasticbeanstalk-fourfront-cgap-system
 metadata_bundles_bucket = elasticbeanstalk-fourfront-cgap-metadata-bundles
 sentry_dsn = ${SENTRY_DSN}
+tibanna_logs_bucket = elasticbeanstalk-fourfront-cgap-tibanna-logs
```

Comment:
Doing this won't break anything as nothing uses the portal as an authoritative source on this bucket, but it should be noted as in other places this bucket name is wrong. This comment applies to all the legacy ini files.